src/rule-transform/rule-content/get-examples-content.ts

```diff
@@ -71,3 +69,57 @@ function getExternalLink(
     `href="${href}">Open in a new tab</a> `
   );
 }
+
+function getAssetsString(assets: TestAssets = {}): string {
+  const sorted = Object.entries(assets).sort((a, b) => comparer(a[0], b[0]));
+
+  if (sorted.length === 0) {
+    return "";
+  }
+
+  const plural = sorted.length === 1 ? ["This", " is"] : ["These", "s are"];
+  const hasJS =
+    sorted.find(([filename]) => filename.endsWith(".js")) !== undefined;
+  const js = hasJS ? "Javascript " : "";
+  const hasCSS =
+    sorted.find(([filename]) => filename.endsWith(".css")) !== undefined;
+  const css = hasCSS ? "CSS " : "";
+  const both = hasJS && hasCSS ? "and " : "";
+
+  const header = `${plural[0]} ${js}${both}${css}file${plural[1]} used in several examples:`;
+
+  const assetsBase = "/WAI/content-assets/wcag-act-rules";
```

Comment:
Otherwise these links don't work in the preview. Had to do the same for links to the examples.

```suggestion
  const assetsBase = "https://w3.org/WAI/content-assets/wcag-act-rules";
```